server/methods/browseChannels.js

```diff
@@ -208,7 +215,7 @@ Meteor.methods({
 			return;
 		}
 
-		if (!['name', 'createdAt', 'usersCount', ['channels', 'teams'].includes(...type) ? ['usernames', 'lastMessage'] : [], ...type === 'users' ? ['username', 'email', 'bio'] : []].includes(sortBy)) {
+		if (!['name', 'createdAt', 'usersCount', ...['channels', 'teams'].includes(type) ? ['usernames', 'lastMessage', 'channelsCount'] : [], ...type === 'users' ? ['username', 'email', 'bio'] : []].includes(sortBy)) {
```

Comment:
Be aware this may conflict depending if this or the other PR touching this line is merged first